lib/ts/recipe/thirdpartyemailpassword/components/themes/signInAndUp/index.tsx

```diff
@@ -41,11 +41,11 @@ const SignInAndUpTheme: React.FC<ThirdPartyEmailPasswordSignInAndUpThemeProps> =
     const usesDynamicLoginMethods = SuperTokens.usesDynamicLoginMethods;
     const dynamicLoginMethods = Multitenancy.getInstanceOrThrow().getLoadedDynamicLoginMethods();
     const thirdPartyEnabled =
-        (props.config.disableEmailPassword === false && usesDynamicLoginMethods === false) ||
+        usesDynamicLoginMethods === false ||
```

Comment:
Why are we not checking the provider count in this case?

Reply:
do you mean like (usesDynamicLoginMethods === false || providers.length > 0) ?

Reply:
do you mean like (usesDynamicLoginMethods === false || providers.length > 0) ?

Reply:
`usesDynamicLoginMethods === false && providers.length > 0`